### ISSUE TITLE
Token acquisition 

### ISSUE THREAD

**Issue by marcoscaceres** [w3c/echidna#92]
It's not clear to me what happens after the token is acquired from W3C staff? Where does the token go?

**Comment by tripu** [comment]
@marcoscaceres, we have work to do to improve documentation about the project, and explain things more clearly.

That being said, the token that you obtain from W3C will be used from that moment on (always the same token) every time you request publication of a a WD. The token is one of the parameters that the Echidna endpoint expects, [as explained here](https://github.com/w3c/echidna/wiki/How-to-use-Echidna#request-a-publication).

Does that answer your question?

**Comment by marcoscaceres** [comment]
ah, ok, got it. Ok, the wiki should say that the token is then POSTed later to the actual labs.... URL.

**Comment by tripu** [comment]
Good idea, @marcoscaceres. [Done](https://github.com/w3c/echidna/wiki/Token-creation) :)

**Comment by marcoscaceres** [comment]
Thanks. Also, there is a slight problem with the process here. It's not straight forward to find out who to contact for a particular group (e.g., I pinged @darobin, and he told me he doesn't have access - then I pinged @dontcallmedom, but he is away until March). Obviously, this kind of thing sucks because it adds significant delays.  

It would be great if there was a form from where to request the token. If a user authenticates against W3C, and provides a unique TR/  shortname, that should be enough to generate and/or retrieve a token, right? Then it could avoid the need to contact a human.

**Comment by plehegar** [comment]
Thank you Marcos. That's food for thoughts. Once we get a bit more confortable with the system, we should reconsider hwo those token get acquired. I'd like to keep this issue opened until then.
